DevGuideExamples/FACE/two-topic/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+cmake_minimum_required(VERSION 3.3...3.27)
+project(opendds_stock CXX)
+enable_testing()
+
+find_package(OpenDDS REQUIRED)
+include(opendds_testing)
+
+# Make sure the MPC-generated headers are gone so the CMake build will use the
+# right ones. This is not needed in a real project.
+# file(GLOB headers "*.h")
+# list(LENGTH headers header_count)
+# if(header_count GREATER 0)
+#   file(REMOVE ${headers})
+# endif()
```

Comment:
This should be uncommented and updated because if I remember correctly, FACE produces `.hpp` files.